rllib/examples/inference/policy_inference_after_training.py

```diff
@@ -147,7 +150,7 @@
         )
     )
 
-    # Create the env to do inference in.
+    # Create a env to do inference in.
```

Comment:
We should bring in here loading the pipelines from checkpoint, too and using them.

Reply:
We have another example, where we do that (the LSTM one, which requires the connector pipeline for a more sophisticated inference loop w/ state in/outs).